controllers/defaults/resources.go

```diff
@@ -93,6 +93,26 @@ var (
 				corev1.ResourceMemory: resource.MustParse("1Gi"),
 			},
 		},
+		"crashcollector": {
+			Requests: corev1.ResourceList{
+				corev1.ResourceCPU:    resource.MustParse("0.5"),
+				corev1.ResourceMemory: resource.MustParse("1Gi"),
+			},
+			Limits: corev1.ResourceList{
+				corev1.ResourceCPU:    resource.MustParse("0.5"),
+				corev1.ResourceMemory: resource.MustParse("1Gi"),
+			},
+		},
+		"exporter": {
```

Comment:
See [this example](https://github.com/rook/rook/blob/master/deploy/charts/rook-ceph-cluster/values.yaml#L344-L349) for exporter resources